addons/hr_holidays/report/hr_leave_report_calendar.py

```diff
@@ -52,7 +52,7 @@ def init(self):
             em.company_id AS company_id,
             em.job_id AS job_id,
             COALESCE(
-                CASE WHEN hl.holiday_type = 'employee' THEN COALESCE(rr.tz, rc.tz) END,
+                COALESCE(rr.tz, rc.tz),
```

Comment:
Can be inlined in the surrounding coalesce.
```suggestion
                rr.tz, rc.tz,
```